code/controllers/subsystems/supply.dm

```diff
@@ -721,3 +721,47 @@ var/datum/controller/subsystem/cargo/SScargo
 			co.partial_shipment_fee = aliquot_shipment_cost
 
 	return 1
+
+/datum/controller/subsystem/cargo/proc/add_junk()
+
```

Comment:
Check if the shuttle is at central when adding the crap.
Would look weird if it would suddenly show up when its already docked at the station

Reply:
Yeah that's for the future. I might just shelf the cargo random event and wait for your cargo update so I can work around that.